### DISCUSSION TITLE
TEAM whats your idea !? !?!

### DISCUSSION THREAD

**Discussion by Lucagamer369** [Chia-Network/chia-blockchain#4895] [General]
MY idea is a Peer list at exporer !!

If you can't get rid of the bug, you should think about a kind of "ADDNODE" implication ... or add a section in explorer where everyone can see current good peers°!!!!

**Comment by gromran** [comment]
If it doesn't find any nodes, an explorer won't find any either. And adding manually is already possible.

**Comment by night-lurker** [comment]
but maybe it should try and connect to 20 peers... and keep looking until it have 20

the other day i was stuck with only 1 peer for 8 hours, until i added 2 more myself
